middlewares/validateCreateProfile.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * This is a validation for creating a profile
+ * @constant
+ *
+ * @param {String} req request object
+ * @param {Object} res response object
+ * @param {Object} err error object
+ *
+ * @returns {Object}
+ *
+ * @exports validateCreateProfile
+ */
+
+const validateCreateProfile = (req, res, next) => {
+  let {
+    firstname, lastname, username, gender, bio, phone, address, image,
+  } = req.body;
+  const urlRegex = /(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9]\.[^\s]{2,})/;
```

Comment:
#### Nitpick

- Have a helper function for this or find a way to encapsulate it out of this file

Reply:
Do you mean the regex?

Reply:
@fejiroofficial , yes the regex